apps/field.go

```diff
@@ -22,6 +23,14 @@ const (
 
 	// A mattermost channel reference (~name).
 	FieldTypeChannel FieldType = "channel"
+
+	TextFieldSubtypeInput     TextFieldSubtype = "input"
```

Comment:
According to https://developers.mattermost.com/integrate/apps/interactivity/ the field is named `text`. Are the docs wrong?

Reply:
@hanzei The field type is `text`, but the text *subtype* can be any that are listed here: https://github.com/mattermost/mattermost-webapp/blob/74cbcb7c3f0dff7e21ecb88776b8841dd1fa55e6/components/widgets/settings/text_setting.tsx#L8

Reply:
It is already documented in the interactivity docs https://github.com/mattermost/mattermost-developer-documentation/blob/master/site/content/integrate/apps/interactivity/_index.md#modal-forms

Reply:
Yep, but it's documented as `text` there not `input`. Fixed in [`3cda372` (#805)](https://github.com/mattermost/mattermost-developer-documentation/pull/805/commits/3cda3721955353179c53fdc4908349e29836a143)